pkg/mdformatter/transformer.go

```diff
@@ -75,6 +78,7 @@ func (t *transformer) Render(w io.Writer, source []byte, node ast.Node) error {
 						if token.Attr[i].Key != "src" {
 							continue
 						}
+						t.sourceCtx.LineNumbers = getLinkLines(source, []byte(token.Attr[i].Val), t.frontMatterLen)
```

Comment:
I am pretty sure we can get this directly from `node (...) offset` 🤔 
Traversing huge file might get us quite latency problems... can we try that? 🤗

Reply:
Sure! Will try to implement this! 🙂

Reply:
I can't seem to find any `offset` method/field which could give line numbers for a particular node. I could only find this [method](https://github.com/yuin/goldmark/blob/master/text/reader.go#L39) in `text` package but functionality seems to be different...

Reply:
It actually shows the offset in source 🤔 usually ;p